downstream/archive/archived-modules/platform/con-inventory-introduction.adoc

```diff
@@ -61,6 +61,9 @@ registry_password='<registry password>'
 
 The first part of the inventory file specifies the hosts or groups that Ansible can work with. 
 
-[NOTE]
+[WARNING]
 ====
-The inventory file variables `registry_username` and `registry_password` are only required if a non-bundle installer is used.
\ No newline at end of file
+You must have valid subscriptions attached before installing the Ansible Automation Platform. If you have a new Red Hat Account, you do not have to perform any additional tasks. By default, Simple Content Access, which hosts access to all your subscriptions and content, is enabled. You can enable Simple Content Access by visiting Red Hat Console. 
+
+The only scenario where you still need to attach a subscription manually is to an air-gapped system that uses Red Hat Satellite for subscription validation. This exception is no longer supported after Red Hat Satellite 16.6.
```

Comment:
There might be a minor typo with the Satellite version. Seems the latest is 6.15. We also have this in the attributes file. 

```suggestion
The only scenario where you still need to attach a subscription manually is with an air-gapped system that uses Red Hat Satellite for subscription validation. This exception is no longer supported after Red Hat Satellite {SatelliteVers}.
```

Reply:
Declining this suggestion, because 16.6 is when support ends. It is supported in the current version 16.5.